README.rst

```diff
@@ -49,39 +49,23 @@ documentation <https://cloud.mbed.com/docs/latest/mbed-cloud-sdk-python/>`__:
 1. Create an API key in the `Pelion Device Management
```

Comment:
The above section can probably be deleted as they are a bit redundant:

These instructions can also be found in the `official
documentation <https://cloud.mbed.com/docs/latest/mbed-cloud-sdk-python/>`__:

Reply:
Removed.